src/Library/demos/Popovers/main.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Popovers",
+  "category": "user_interface",
+  "description": "Popups are used to display content in a container above all other content on the window",
+  "panels": [
+    "code",
+    "preview"
```

`ui` is more important here

```suggestion
    "ui",
    "preview"
```